model/project_ref_test.go

```diff
@@ -2961,6 +2961,27 @@ func TestSaveProjectPageForSection(t *testing.T) {
 	}
 	_, err = SaveProjectPageForSection("iden_", update, ProjectPageGeneralSection, false)
 	assert.NoError(err)
+
+	// Test successful external link update
+	update = &ProjectRef{
+		ExternalLinks: []ExternalLink{
+			{URLTemplate: "https://arnars.com/{version_id}", DisplayName: "A link"},
+		},
+	}
+	_, err = SaveProjectPageForSection("iden_", update, ProjectPagePluginSection, false)
```

Comment:
Can we please add a test that checks that the `{version_id}` was replaced successfully, validating that the url is equal to the url we expect it to be?